packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -480,7 +478,7 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
   double _getIntrinsicSize({
     Axis sizingDirection,
     double extent, // the extent in the direction that isn't the sizing direction
-    _ChildSizingFunction childSize // a method to find the size in the sizing direction
+    double Function(RenderBox child, double extent) childSize, // a method to find the size in the sizing direction
```

Comment:
similarly here. the very name of the typedef is documentation. I don't see the value of inlining the type.

Reply:
One advantage is that you don't have to jump in the source to see what the real signature is.

Perhaps renaming `childSize` to `getChildSize`?